### ISSUE TITLE
Feature Request: Account Migration

### ISSUE THREAD

**Issue by brecla** [oxen-io/session-desktop#1972]
Since every Session ID corresponds to a private key that should be changed after some time (usually a couple of years) to be a moving target rather that a static one to potential attackers, there should be a simple and straightforward procedure available to end users to accomplish this task. 

There should be some kind of status-message which acts as a reference to a new Session ID. It must be made sure that it can be proven that the migration notice is authored by someone with access to both keys, the old one and the new one.

**Comment by peepopoggers** [comment]
Agreed on this. Maybe the Recovery Phrase could be migrated to a new ID or something

**Comment by Bilb** [comment]
> I agree on this. Maybe the Recovery Phrase could be migrated to a new ID or something or you could skip the extra step and maybe you could just straight up directly migrate. Would be a nice feature.

this is not going to work with Session as the recovery phrase derives the session id.


> There should be some kind of status-message which acts as a reference to a new Session ID. It must be made sure that it can be proven that the migration notice is authored by someone with access to both keys, the old one and the new one.

You are basically asking for a "I've changed my SessionID message to this SessionID" message, automatically sent to all your contacts when you press a button?

**Comment by peepopoggers** [comment]
it would be nice to have an export and import feature for contacts at least so transferring across accounts is easier
